.github/workflows/clang_tidy.yml

```
@@ -0,0 +1,46 @@
+name: 🧹 clang-tidy
+
+on: [push, pull_request]
+
+jobs:
+  run_clang_tidy:
+    name: clang-tidy
+    runs-on: ubuntu-22.04
+    env: {CXXFLAGS: "-Werror -Wshadow -Woverloaded-virtual -Wunreachable-code -fsanitize=address -fsanitize=undefined"}
+    steps:
+    - uses: actions/checkout@v4
+    - name: install dependencies
+      run: .github/workflows/dependencies/clang15.sh
+    - name: Build & run clang-tidy
+      run: |
+
+        export CXX=$(which clang++-15)
+        export CC=$(which clang-15)
+
+
+        # The following wrapper ensures that only source files
+        # in WarpX/Source/* are actually processed by clang-tidy
```

Should this be updated?
```suggestion
        # The following wrapper ensures that only source files
        # in multi_physics/QED* are actually processed by clang-tidy
```
Not sure whether this is the right suggestion.

---

You are right! I copy-pasted that without paying attention!